suite-common/wallet-utils/src/sendFormUtils.ts

```diff
@@ -474,3 +475,6 @@ export const getExcludedUtxos = ({
     });
     return excludedUtxos;
 };
+
+export const getLamportsFromSol = (amountInSol: string) =>
```

Comment:
There was some discussion between @PeterBenc and I about whether this util should be placed here or in `sendFormSolanaActions`, what are your thoughts?

Reply:
If its used multiple times, I am fine with having this here, no need to raise this with trezor guys :+1: